src/goRunTestCodelens.ts

```diff
@@ -85,7 +85,7 @@ export class GoRunTestCodeLensProvider extends GoBaseCodeLensProvider {
 					buildFlags.push(`"${vsConfig['buildTags']}"`);
```

Comment:
Since this is related to buildtags specifically and not buildflags in general, shouldnt we do the join that you did on line 85 instead?

Reply:
Referring to [usage](https://github.com/Microsoft/vscode-go/blob/dd927509a3db3833dc02103e1cf6290074f56a4b/src/debugAdapter/goDebug.ts#L297) and [interface declaration](https://github.com/Microsoft/vscode-go/blob/dd927509a3db3833dc02103e1cf6290074f56a4b/src/debugAdapter/goDebug.ts#L138), it seems to assume a `string?`, but is this reference place wrong(or not just Delve class)?